tests/run_tests.sh

```diff
@@ -3,8 +3,7 @@
 cd $(dirname $0)
 
 if [ "$RUNCOVERAGE" == "1" ]; then
-    python3 -m coverage run --source ctre -m pytest "$@"
-    python3 -m coverage report -m
+    python3 -m pytest --cov-report html --cov ctre "$@"
```

Comment:
I'm all for using pytest-cov, but you need to:
* make the report readable from Travis CI, so let it report to the terminal, and
* install pytest-cov on Travis CI.